fastlane_bot/events/utils.py

```diff
@@ -638,8 +639,8 @@ def get_config(
     cfg.LIMIT_BANCOR3_FLASHLOAN_TOKENS = limit_bancor3_flashloan_tokens
     cfg.DEFAULT_MIN_PROFIT_GAS_TOKEN = Decimal(default_min_profit_gas_token)
     cfg.GAS_TKN_IN_FLASHLOAN_TOKENS = (
-        cfg.NATIVE_GAS_TOKEN_ADDRESS in flashloan_tokens
-        or cfg.WRAPPED_GAS_TOKEN_ADDRESS in flashloan_tokens
+        cfg.NATIVE_GAS_TOKEN_ADDRESS.lower() in flashloan_tokens.lower()
+        or cfg.WRAPPED_GAS_TOKEN_ADDRESS.lower() in flashloan_tokens.lower()
     )
```

Comment:
Since the w3 module is required for checksum, and the cfg is required to be initialized before using w3, its not as clean to checksum the flashloan_tokens AND target_tokens at this point.
i.e. if we want to checksum flashloan tokens here it can go after w3 intialized but then we also need to pass the target tokens - when we have specific functions called handle_flashloan_tokens() etc.
So if the native or wrapped gas token are not  checksummed we can check for their presence with lower() instead without modifying